lib/ansible/modules/windows/win_product_facts.ps1

```diff
@@ -0,0 +1,48 @@
+#!powershell
+# This file is part of Ansible
+
+# Copyright: (c) 2017, Dag Wieers (dagwieers) <dag@wieers.com>
```

Comment:
It's 2018 now :)

Reply:
The code originated from 2017 ;-)

Reply:
Just living the the past, gotta let it go :)